azure-pipelines.yml

```diff
@@ -1,7 +1,10 @@
 trigger:
   branches:
     include:
-    - '*'
+    - 'devel'
+    - 'version-1-6'
+    - 'version-1-2'
+    - 'version-1-0'
```

Comment:
Can this be modified to something like
```suggestion
    - 'version-*
```
so that future versions are covered?

Reply:
Should work